projects/avodata/requirements.txt

```diff
@@ -0,0 +1,50 @@
+altgraph==0.10.2
```

Comment:
This contains many libraries we're not using. I guess `freeze` wasn't a good idea.
I think you only need `numpy==1.8.0rc1`.

You can use `virtualenv`:
https://virtualenv.pypa.io/en/latest/
To create an isolated environment, and then see which dependencies you actually need.

BTW, how did you install all these libraries?